src/platform/tgatexture.cpp

```diff
@@ -93,7 +93,7 @@ detail::TextureImpl* Load( File& F )
         Succ = F.Read( Buf, InSize );
         if( Succ )
         {
-            detail::TextureImpl::ConvertRGBtoRGBA( ( uint8_t* )( void* )Buf.c_str(), ImgSize, &impl->mData.at( 0 ) );
```

Comment:
Ah disregard my comment, I see no better way for Width() and Height().
So the question is: a) can we suppress warnings b) is it worth fixing it's a bit less readable?